src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -26,6 +26,8 @@
 #include <credentials/PersistentStorageOpCertStore.h>
 #include <crypto/DefaultSessionKeystore.h>
 #include <errno.h>
+#include <gtest/gtest.h>
+#include <ios>
```

Comment:
Do we need the ios header? Generally we try to avoid that on things that may run on embedded. It seems not needed before.

Reply:
Fixed